src/tiledb/cloud/vcf/query.py

```diff
@@ -327,6 +331,7 @@ def build_read_dag(
     :param log_uri: log array URI for profiling, defaults to None
     :param namespace: TileDB-Cloud namespace, defaults to None
     :param resource_class: TileDB-Cloud resource class for UDFs, defaults to None
+    :param batch_resources: TileDB-Cloud resource class for batch UDFs, defaults to None
```

Comment:
Can can use `resources` instead of `batch_resources`? That'd make it consistent with other APIs.